tst/database/parse-server.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import { DatabaseConfig } from "../../src/database/database-config";
+import ParseServer from "../../src/database/parse-server";
+
+describe("ParseServer Tests", () => {
+  let ParseMock: any;
+  let database: any;
+
+  beforeAll(() => {
+    ParseMock = jest.createMockFromModule("parse");
+    database = ParseServer.getInstance(ParseMock, null);
+  });
+
+  it("initializing ParseServer", async () => {
+    const database2 = ParseServer.getInstance(ParseMock, null);
+    expect(database).toBe(database2);
```

Comment:
This is to check if the two calls to `getInstance()` return the same instance. They should, because ParseServer is a Singleton. This is also why the below calls are expected to only have been called once.

Reply:
wouldn't we want the check to be `expect(database2).toBe(database);` ? its very nit picky but technically local variables should be the focus of the test. also can we change database2 to actualDatabase so it reads `expect(actualDatabase).toBe(database);`

Reply:
Yea that makes sense, will update